src/execution/mod.rs

```diff
@@ -193,8 +193,8 @@ impl CallInfo {
         let storage_entries = self
             .accessed_storage_keys
             .into_iter()
-            .map(|key| (self.contract_address.clone(), key))
-            .collect::<HashSet<(Address, ClassHash)>>();
+            .map(|key| (self.contract_address.clone(), key.0))
+            .collect::<HashSet<(Address, [u8; 32])>>();
```

Comment:
Why did we change this type?

Reply:
We are returning a StorageEntry hash map, there is a to do in that struct pointing that the second field should be a field. I agree we should normalise that value but it is unclear